tower/Cargo.toml

```diff
@@ -67,7 +67,6 @@ hdrhistogram = { workspace = true, optional = true }
 indexmap = { workspace = true, optional = true }
 slab = { workspace = true, optional = true }
 tokio = { workspace = true, features = ["sync"], optional = true }
-tokio-stream = { workspace = true, optional = true }
```

Comment:
This is a breaking change, since optional dependencies that aren't mentioned as `dep:that-dependency` in a feature create an implicit feature. Can you add a feature of the same name with a comment that it's a no-op and will be removed in the future?